requirements.txt

```diff
@@ -11,3 +11,4 @@ sncosmo
 colossus
 uncertainties
 kcorrect
+webbpsf
```

Comment:
just checking, this is easy installable and being used, or otherwise could also be moved to the test_requirements?

Reply:
webbpsf does not have restrictions on operating system so there is no need to move it to test_requirements. It can also be imported with no problems. The only thing is that in order to actually use the functions and classes within the module, external files are needed. If the user tries to use webbpsf features without these external files, webbpsf has its own very beautiful error message that links the user to the files.